inc/class-wpseo-replace-vars.php

```diff
@@ -1210,52 +1210,59 @@ private function add_custom_field_prefix( $field ) {
-			'pagetotal'                 => __( 'Replaced with the current page total', 'wordpress-seo' ),
-			'pagenumber'                => __( 'Replaced with the current page number', 'wordpress-seo' ),
-			'caption'                   => __( 'Attachment caption', 'wordpress-seo' ),
-			'focuskw'                   => __( 'Replaced with the posts focus keyword', 'wordpress-seo' ),
-			'term404'                   => __( 'Replaced with the slug which caused the 404', 'wordpress-seo' ),
-			'cf_<custom-field-name>'    => __( 'Replaced with a posts custom field value', 'wordpress-seo' ),
-			'ct_<custom-tax-name>'      => __( 'Replaced with a posts custom taxonomies, comma separated.', 'wordpress-seo' ),
-			'ct_desc_<custom-tax-name>' => __( 'Replaced with a custom taxonomies description', 'wordpress-seo' ),
+		$replacement_variables = array(
+			new WPSEO_Replacement_Variable( 'pt_single', __( 'Post type (singular)', 'wordpress-seo' ), __( 'Replaced with the content type single label', 'wordpress-seo' ) ),
+			new WPSEO_Replacement_Variable( 'pt_plural', __( 'Post type (plural)', 'wordpress-seo' ), __( 'Replaced with the content type plural label', 'wordpress-seo' ) ),
+			new WPSEO_Replacement_Variable( 'modified', __( 'Modified', 'wordpress-seo' ), __( 'Replaced with the post/page modified time', 'wordpress-seo' ) ),
+			new WPSEO_Replacement_Variable( 'id', __( 'ID',  'wordpress-seo' ), __( 'Replaced with the post/page ID', 'wordpress-seo' ) ),
+			new WPSEO_Replacement_Variable( 'name', __( 'Name', 'wordpress-seo' ), __( 'Replaced with the post/page author\'s \'nicename\'', 'wordpress-seo' ) ),
+			new WPSEO_Replacement_Variable( 'user_description', __( 'User description', 'wordpress-seo' ), __( 'Replaced with the post/page author\'s \'Biographical Info\'', 'wordpress-seo' ) ),
+			new WPSEO_Replacement_Variable( 'page', __( 'Page number',  'wordpress-seo' ), __( 'Replaced with the current page number with context (i.e. page 2 of 4)', 'wordpress-seo' ) ),
+			new WPSEO_Replacement_Variable( 'pagetotal', __( 'Pagetotal', 'wordpress-seo' ), __( 'Replaced with the current page total', 'wordpress-seo' ) ),
+			new WPSEO_Replacement_Variable( 'pagenumber', __( 'Pagenumber', 'wordpress-seo' ), __( 'Replaced with the current page number', 'wordpress-seo' ) ),
+			new WPSEO_Replacement_Variable( 'caption', __( 'Caption', 'wordpress-seo' ), __( 'Attachment caption', 'wordpress-seo' ) ),
+			new WPSEO_Replacement_Variable( 'focuskw', __( 'Focus keyword',  'wordpress-seo' ), __( 'Replaced with the posts focus keyword', 'wordpress-seo' ) ),
+			new WPSEO_Replacement_Variable( 'term404', __( 'Term404', 'wordpress-seo' ), __( 'Replaced with the slug which caused the 404', 'wordpress-seo' ) ),
+			new WPSEO_Replacement_Variable( 'cf_<custom-field-name>', __( '<custom-field-name> (custom field)', 'wordpress-seo' ) , __( 'Replaced with a posts custom field value', 'wordpress-seo' ) ),
```

Comment:
I am also uncertain on how these will come out of a translation.

Reply:
I don't think you want to translate custom field names or custom taxonomies. The users often enter these themselves, and we cannot be sure what language they used. I also imagine it being weird when they enter
products and product_prices we will translate to producten and product_prices, which is strange.

Reply:
nvm I am a silly man